webapp/components/survey/CategorySelector/CategorySelector.js

```diff
@@ -13,29 +12,20 @@ import { useSurvey } from '@webapp/store/survey'
 import Dropdown from '@webapp/components/form/Dropdown'
 import PanelRight from '@webapp/components/PanelRight'
 import CategoryList from '@webapp/views/App/views/Designer/CategoryList'
```

Comment:
in another pull request, please move CategoryList under @webapp/components/survey

Reply:
sure